### DISCUSSION TITLE
iframe 样式疑问

### DISCUSSION THREAD

**Discussion by HonorKnight8** [HEIGE-PCloud/DoIt#183] [General]
我想在 index.md 里面写一个 iframe 

`<iframe src="/study_and_practice/"
        width="100%"
        height="100%"
        frameborder="no"
        border="0"
        marginwidth="0"
        marginheight="0"
        scrolling="no"
        allowtransparency="yes"></iframe>`

src 是自己另外写的 JavaWeb 页面

这样能不能实现 iframe 里的内容，直接使用主题现有的样式？

**Comment by HEIGE-PCloud** [comment]
首先使用 `hugo` 生成静态页面，然后可以在 `public/css/` 文件夹下找到 `style.min.css`，其中包含主题使用的所有 CSS 样式。

然后在 iframe 的页面中插入该 CSS 文件  `<link type="text/css" href="style.min.css" />` 并保证页面内元素遵循相应的规则即可使用主题样式。
